setup.py

```diff
@@ -45,7 +45,7 @@
 #
 # Other demos do not have specific requirements
 EXTRAS_REQUIRES = {
-    "demos": ["numba", "jupyter", "seaborn", "rdflib", "mplleaflet==0.0.5"],
+    'demos': ["numba", "jupyter", "seaborn", "rdflib", "mplleaflet==0.0.5"],
```

Comment:
was this not being checked by black in the past?

Reply:
It is being checked, this is one of the changes I made to verify that the new CI is flagging things correctly; I'll revert it before landing.